src/routes/OrganizationsList/OrganizationsListTableData.tsx

```diff
@@ -0,0 +1,187 @@
+    let count = null;
+    if (!props.user) {
+      fetchOrg(props.orgName)
+        .then((data) => {
+          count = data?.teams ? Object.keys(data?.teams)?.length : 0;
+          setTeamCount(count);
+        })
+        .catch((err) => {
+          console.error(err);
+          count = 'N/A';
+          setTeamCount(count);
+        });
+    } else {
+      count = 'N/A';
+      setTeamCount(count);
+    }
+  }, [props.orgName]);
+
+  return <>{teamCount !== null ? teamCount : <Skeleton width="100%" />}</>;
+}
+
+function MemberCount(props: MemberCountProps) {
+  const [memberCount, setMemberCount] = useState<string | number>(null);
+
+  useEffect(() => {
+    let count = null;
+    if (!props.user) {
+      fetchMembersForOrg(props.orgName)
+        .then((data) => {
+          count = data.length;
+          setMemberCount(count);
+        })
+        .catch((err) => {
+          console.error(err);
+          count = 'N/A';
+          setMemberCount(count);
+        });
+    } else {
+      count = 'N/A';
+      setMemberCount(count);
+    }
+  }, [props.orgName]);
+
+  return <>{memberCount !== null ? memberCount : <Skeleton width="100%" />}</>;
+}
+
+function RobotCount(props: RobotCountProps) {
+  const [robotCount, setRobotCount] = useState<string | number>(null);
+
+  useEffect(() => {
+    let count = null;
+    fetchRobotsForNamespace(props.orgName)
+      .then((data) => {
+        count = data.length;
+        setRobotCount(count);
+      })
+      .catch((err) => {
+        console.error(err);
+        count = 'N/A';
+        setRobotCount(count);
+      });
+  }, [props.orgName]);
+
+  return <>{robotCount !== null ? robotCount : <Skeleton width="100%" />}</>;
+}
+
+function RepoCount(props: RepoCountProps) {
+  return <>{props.count !== null ? props.count : <Skeleton width="100%" />}</>;
+}
+
+function RepoLastModifiedDate(props: RepoLastModifiedDateProps) {
+  return (
+    <>
+      {props.lastModifiedDate !== 0 ? (
+        formatDate(props.lastModifiedDate)
+      ) : (
+        <Skeleton width="100%" />
+      )}
+    </>
+  );
+}
+
+export default function OrgTableData(props: OrgTableProps) {
+  const [repoCount, setRepoCount] = useState<string | number>(null);
+  const [lastModifiedDate, setLastModifiedDate] = useState<number>(0);
+  const getLastModifiedRepoTime = (repos: IRepository[]) => {
+    // get the repo with the most recent last modified
+    if (!repos || !repos.length) {
+      return -1;
+    }
+
+    const recentRepo = repos.reduce((prev, curr) =>
+      prev.last_modified < curr.last_modified ? curr : prev,
+    );
+    return recentRepo.last_modified;
+  };
+
+  useEffect(() => {
+    let count = null;
+    let lastModified = null;
+    fetchRepositoriesForNamespace(props.orgName)
+      .then((data) => {
+        count = data.length;
+        lastModified = getLastModifiedRepoTime(data);
+        setRepoCount(count);
+        setLastModifiedDate(lastModified);
+      })
+      .catch((err) => {
```

Comment:
should we display an error here?

Reply:
That's a tough one since the cell only has so much space. Maybe it should show just the text `Error`?